src/components/Graph.react.js

```diff
@@ -271,6 +279,12 @@ const graphPropTypes = {
      */
     relayoutData: PropTypes.object,
 
+    /**
+     * Data from latest restyle event which occurs
+     * when the user toggles a legend item
```

Comment:
It's not obvious to me if this prop is read-only or not. From reading the code I'd say changing it has no effect, so it's a read only and should be stated.